### DISCUSSION TITLE
Why is there an svg picture in red in the github repository saying "failing" when it is NOT "failing"?

### DISCUSSION THREAD

**Discussion by davidhe1986** [wingtk/gvsbuild#1256] [General]
I slightly modified the file "builder.py" for Windows 11 PRO and the script is building the library gtk 3.0 and 4.0 as expected!
I am confused why is the svg not green saying "working"?
![Screenshot 2024-02-25 204927](https://github.com/wingtk/gvsbuild/assets/69299053/514099b7-e0c6-4a95-8fea-71dc0de1db81)

**Comment by danyeaw** [comment]
Hi @davidhe1986, thanks for pointing this out. I submitted a PR to fix this: https://github.com/wingtk/gvsbuild/pull/1260

**Reply by davidhe1986** [reply]
I updated my request with a picture.
